packages/react/src/theme/default-theme.ts

```diff
@@ -17,40 +17,47 @@
  */
 
 import {experimental_extendTheme as extendTheme} from '@mui/material/styles';
-import {Theme} from '../models';
+import lightTokens from '@oxygen-ui/primitives/dist/design-tokens/web/oxygen/es/tokens';
+import darkTokens from '@oxygen-ui/primitives/dist/design-tokens/web/oxygen/es/dark.tokens';
+import {RecursivePartial, Theme} from '../models';
 
-const defaultTheme: Theme = extendTheme({
+export const DEFAULT_THEME_OPTIONS: RecursivePartial<Theme> = {
   colorSchemes: {
     dark: {
       palette: {
         primary: {
-          main: '#FF7300',
+          main: darkTokens.OxygenOxygenColorsPrimaryDefault,
         },
       },
     },
     light: {
       palette: {
         primary: {
           contrastText: '#fff',
-          main: '#FF7300',
+          main: lightTokens.OxygenOxygenColorsPrimaryDefault,
         },
         secondary: {
-          contrastText: '#40404B',
-          main: '#F7F8FB',
+          // TODO: Need a dedicated variable for secondary button text.
```

Comment:
@savindi7 Please attend to the `TODO` items listed here and get them sorted by the design team.